repositories.yaml

```diff
@@ -180,6 +180,25 @@ applications:
       - v1_name: firefox-desktop
         app_id: firefox.desktop
 
+  - app_name: firefox_desktop_background_updater
```

Comment:
My preference would be to drop the trailing `r` everywhere for consistency.

For the canonical_app_name, we could say "Firefox for Desktop Background Update Task".

Reply:
This matches :nalexander's comment in the bug. https://bugzilla.mozilla.org/show_bug.cgi?id=1704878#c4